packages/web-react/src/components/Box/README.md

```diff
@@ -0,0 +1,82 @@
+# Box
+
+The Box component is a simple container around content or other components.
+
+```jsx
+<Box>{/* Content go here */}</Box>
+```
+
+## Border
+
+You can define border width, color, and radius using the `borderColor`, `borderRadius`, and `borderWidth` props.
+
+```jsx
+<Box borderColor="basic" borderRadius="200" borderWidth="100">
+  {/* Content go here */}
+</Box>
+```
+
+## Padding
+
+You can define padding using the `padding` prop.
+
+```jsx
+<Box padding="space-200">{/* Content go here */}</Box>
+```
+
+It is also possible to define padding for horizontal and vertical sides using the `paddingX` and `paddingY` props.
+
+```jsx
+<Box paddingX="space-200" paddingY="space-300">
+  {/* Content go here */}
+</Box>
+```
+
+You can also define padding for each side using the `paddingTop`, `paddingRight`, `paddingBottom`, and `paddingLeft` props.
+
+```jsx
+<Box paddingTop="space-200" paddingRight="space-300" paddingBottom="space-400" paddingLeft="space-500">
+  {/* Content go here */}
+</Box>
+```
+
+Responsive values can be set for each prop using an object:
+
+```jsx
+<Box padding={{ mobile: 'space-200', tablet: 'space-300', desktop: 'space-400' }}>{/* Content go here */}</Box>
+```
+
+## Background Color
+
+You can define background color using the `backgroundColor` prop.
+
+```jsx
+<Box backgroundColor="basic">{/* Content go here */}</Box>
```

Comment:
Do we have `basic` color for backgrounds?

Reply:
Mistake. Thanks. Corrected to primary.